cli/commands/provision/tui.py

```
@@ -0,0 +1,327 @@
+    
+    @work(exclusive=True, thread=True)
+    def provision_instance(
+        self,
+        name: str,
+        zone: str,
+        machine_type: str,
+        swap_gb: int,
+        optimize_dnf: bool,
+    ) -> None:
+        """Run the provisioning in a background thread."""
+        worker = get_current_worker()
```

Ruff (0.14.13)

Local variable `worker` is assigned to but never used

Remove assignment to unused variable `worker`

(F841)